src/merkle/imr.rs

```diff
@@ -58,14 +58,6 @@ impl CommitMerkleRoot for Imr {
 }
 
 impl Imr {
-    /// The IV used in the second pass of IMR computation.
-    fn pass_two_iv<J: Jet>(node: &CommitNodeInner<J>) -> Self {
-        match node {
-            CommitNodeInner::Hidden(_) => Imr::tag_iv(b"Simplicity-Draft\x1fHidden"),
-            _ => Imr::tag_iv(b"Simplicity-Draft\x1fIdentity"),
-        }
-    }
```

Comment:
397cf75d4890f31fff27aa93edd3f619c3869b64: It is really surprising that the code works without the pass two IV. Are we running tests against the C code?

Reply:
`decode::assert_lr` assets the same IMR for `master`. It seems that pass two of the IMR doesn't use the children IMRs, but just the children TMRs. Strange, but good find.

Reply:
Heh, yeah, I was surprised too. And yes, I did check it against the C code when I first noticed this.